src/renderer/views/Watch/Watch.js

```diff
@@ -1087,6 +1087,11 @@ export default defineComponent({
       }
 
       const nextVideoInterval = this.defaultInterval
+      if (this.watchingPlaylist && this.$refs.watchVideoPlaylist.shouldStopDueToPlaylistEnd) {
+        // Let `watchVideoPlaylist` handle end of playlist, no countdown needed
+        this.$refs.watchVideoPlaylist.playNextVideo()
+        return
+      }
```

Comment:
As this variable is not needed for this if, it could be moved below it so it only gets created when it's actually needed.
```suggestion
      if (this.watchingPlaylist && this.$refs.watchVideoPlaylist.shouldStopDueToPlaylistEnd) {
        // Let `watchVideoPlaylist` handle end of playlist, no countdown needed
        this.$refs.watchVideoPlaylist.playNextVideo()
        return
      }
      const nextVideoInterval = this.defaultInterval
```